tools/hyphy/compile.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+EM_FLAGS="-sSTACK_SIZE=2097152 -g4 -sASSERTIONS=1"
```

Comment:
Once you're done testing, it's worth replacing `-g4` with `-O2` so Emscripten produces smaller .wasm/.js files

Reply:
Also, which version of Emscripten are you using? I'm getting `Attempt to set a non-existent setting: STACK_SIZE`. It's best to use the Emscripten image mentioned in https://github.com/biowasm/biowasm/blob/main/CONTRIBUTING.md

Reply:
I'm using version 3.1.44. Unfortunately, I do get the error 

```
10:32:46 sweaver@Stevens-MacBook-Pro.local build hyphy ? node ./hyphy.js 

Stack overflow detected.  You can try increasing -sSTACK_SIZE (currently set to 65536)
/Users/sweaver/Programming/biowasm/tools/hyphy/build/hyphy.js:129
      throw ex;
      ^
```

If it is not set. 

In emscriptem 2, it appears the analogous flag is `EMFLAGS="-sTOTAL_STACK=2097152"`.